recipes/systemc-cci/1.0.0/CMakeLists.txt

```
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(cci)
+
+set(LIBRARY_VERSION_MAJOR 1 CACHE STRING "major version" FORCE)
+set(LIBRARY_VERSION_MINOR 0 CACHE STRING "minor version" FORCE)
+set(LIBRARY_VERSION_PATCH 0 CACHE STRING "minor version" FORCE)
+set(LIBRARY_VERSION ${LIBRARY_VERSION_MAJOR}.${LIBRARY_VERSION_MINOR}.${LIBRARY_VERSION_PATCH} CACHE STRING "version" FORCE)
+
+set(LIBRARY_NAME cciapi)
+
+set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
+
+set(POSITION_INDEPENDENT_CODE ON)
+
+include(${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(KEEP_RPATHS)
```
better compatibility on Mac

---

Mac build is currently not supported